ts/mixpanel.ts

```diff
@@ -54,6 +66,13 @@ export const terminateMixpanel = () => {
   }
 };
 
+export const isMixpanelInitialized = () => mixpanel != null;
```

Comment:
Would it make sense to rename this function to avoid confusion with the similarly named [store property](https://github.com/pagopa/io-app/blob/1ecabda7c7424d4c52d92e835d97e81fcf42689a/ts/features/mixpanel/store/reducer/index.ts#L10)? What do you think?

Reply:
Yep, done it in 34270372e2eb4374d01d245dd47f834698558df7